docbrown/db/src/vertex.rs

```diff
@@ -0,0 +1,171 @@
+use crate::edge::EdgeView;
+use crate::view_api::internal::GraphViewInternalOps;
+use crate::view_api::{VertexListOps, VertexViewOps};
+use docbrown_core::tgraph::VertexReference;
+use docbrown_core::{Direction, Prop};
+use std::collections::HashMap;
+use std::sync::Arc;
+
+pub struct VertexView<G: GraphViewInternalOps> {
+    //FIXME: Not sure Arc is good here, maybe this should just own a graph and rely on cheap clone...
+    graph: Arc<G>,
```

Comment:
Depends what graph do you expect to have here
`Graph` in db is already clone friendly so you can own it, TemporalGraph not so much